frontend/src/components/results/PlayerResultsItem.tsx

```diff
@@ -129,13 +153,13 @@ function PlayerResultsItem(props: PlayerResultsCardProps) {
       </PlayerContent>
 
       <td>
-        <Text>{getScore(bestSubmission)}</Text>
+        <Text>{getScore()}</Text>
       </td>
       <td>
-        <Text>{getSubmissionTime(bestSubmission, gameStartTime)}</Text>
+        <Text>{getSubmissionTime()}</Text>
       </td>
       <td>
-        <Text>{getSubmissionCount(player)}</Text>
+        <Text>{getSubmissionCount()}</Text>
```

Comment:
@alankbi, I think you made this change. Correct me if I'm wrong.

Reply:
I don't remember making this change, but looking through the codebase I think it's because these methods are newly added, as opposed to before when using the exported functions from Utility

Reply:
Resolving this because we discussed it on the call.